src/plugins_manager.coffee

```diff
@@ -21,6 +21,8 @@ define [
     @param [Object, String] data Any additional data required by the plugins
     ###
     notify: (action, data) ->
+      return unless @session?.shop_code
```

Comment:
I believe this is redundant. We should ✂️ . `notify` will [never be called](https://github.com/skroutz/analytics.js/blob/master/src/session.coffee#L40) due to the [session connect safe guard](https://github.com/skroutz/analytics.js/pull/83/commits/098e21ee87748fa7ee43ae0fed96abab6391a3f1#diff-620774fd73e3a52fa36c5c3beac74b331e6b3491e3d28a6702eab919ed0a63eaR33).

Reply:
done!